test/browserbehavior/DefaultBrowserBehavior.test.ts

```diff
@@ -122,41 +127,30 @@ describe('DefaultBrowserBehavior', () => {
       expect(new DefaultBrowserBehavior().majorVersion()).to.eq(13);
       expect(new DefaultBrowserBehavior().requiresBundlePolicy()).to.eq('max-bundle');
       expect(new DefaultBrowserBehavior().getDisplayMediaAudioCaptureSupport()).to.be.false;
-      const enableUnifiedPlan = true;
-      expect(
-        new DefaultBrowserBehavior({
-          enableUnifiedPlanForChromiumBasedBrowsers: enableUnifiedPlan,
-        }).requiresUnifiedPlan()
-      ).to.eq(true);
-      expect(
-        new DefaultBrowserBehavior({
-          enableUnifiedPlanForChromiumBasedBrowsers: enableUnifiedPlan,
-        }).requiresUnifiedPlanMunging()
-      ).to.eq(true);
-      expect(
-        new DefaultBrowserBehavior({
-          enableUnifiedPlanForChromiumBasedBrowsers: !enableUnifiedPlan,
-        }).requiresUnifiedPlan()
-      ).to.eq(true);
-      expect(
-        new DefaultBrowserBehavior({
-          enableUnifiedPlanForChromiumBasedBrowsers: !enableUnifiedPlan,
-        }).requiresUnifiedPlanMunging()
-      ).to.eq(true);
```

Comment:
Unit test for requiresUnifiedPlan() is removed here as `RTCRtpTransceiver.prototype.hasOwnProperty('currentDirection');` will always be false here